test/msw-api/setup-server/response-patching.test.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @jest-environment node
+ */
+import fetch from 'node-fetch'
+import { rest } from 'msw'
+import { setupServer } from 'msw/node'
+
+const server = setupServer(
+  rest.get('https://test.mswjs.io/user', async (req, res, ctx) => {
+    const originalResponse = await ctx.fetch(
+      'https://api.github.com/users/octocat',
```

Comment:
GitHub API has rate limiting that may result into flaky CI. I suggest to use `https://httpbin.org/get`, which doesn't seem to have one (afaik).